src/components/create-user.html

```diff
@@ -1,95 +1,159 @@
-                        <input type="text" id="email" name="email" placeholder="Your email..">
-                        <div>
-                            <button class="button">Create</button>
-                        </div>
-                    </form>
-                </div>
-            </div>
+  <template>
+    <style>
+      :host {
+        display: block
+      }
+
+      .div-center {
+        text-align: center;
+      }
+
+      input[type=text],
+      select {
+        width: 100%;
+        padding: 12px 20px;
+        margin: 8px 0;
+        display: inline-block;
+        border: 1px solid #87CEFA;
+        border-radius: 4px;
+        box-sizing: border-box;
+      }
+
+      div {
+        border-radius: 5px;
+        background-color: #f2f2f2;
+        padding: 20px;
+      }
+
+      .border {
+        border: #666 1px solid;
+        width: 20%;
+        margin: auto;
+        margin-top: 25px;
+        padding-bottom: 25px;
+      }
+
+      .button {
+        background-color: #87CEFA;
+        border: none;
+        color: white;
+        padding: 5px 12px;
+        text-align: center;
+        text-decoration: none;
+        display: block;
+        margin: auto;
+        font-size: 16px;
+        margin-top: 25px;
+      }
+
+      button:hover {
+        background-color: rgb(196, 226, 245);
+      }
+    </style>
+
+    <h1 class="div-center">Create User</h1>
+    <div>
+      <div class="container">
+        <div class="form-border">
+          <form>
+            <label for="fname">First Name</label>
+            <input type="text" id="fname" name="firstname" placeholder="Your name..">
+
+            <label for="lname">Last Name</label>
+            <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+
+            <label for="phone">Phone</label>
+            <input type="text" id="phone" name="phone" placeholder="Your phone number..">
+
+            <label for="email">Email</label>
+            <input type="text" id="email" name="email" placeholder="Your email..">
+          </form>
+          <div>
+            <button class="button" on-click="createUser">Create User</button>
+            <button class="button" on-click="deleteAllUsers">Delete All Users</button>
+          </div>
         </div>
-    </template>
+      </div>
+    </div>
+  </template>
+
+  <script type="module">
+    import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    import { LocalStorage } from '../api/localStorage.js';
+    import faker from '../../node_modules/faker';
+
+    let LocalUsers = new LocalStorage();
+    
+    let userEntry = {
+      '_id': Math.floor(10000 + Math.random() * 9000),
+      'name': faker.fake('{{name.firstName}} {{name.lastName}}')
+    };
+
+    class CreateUser extends PolymerElement {
+      static get is() { return 'create-user'; }
+      constructor() {
+        super();
+      }
+      
+      createUser() {
+        LocalUsers.createUser(userEntry);
```

Comment:
~Where is `userEntry` coming from?~ You should use the data from your inputs to populate the `userEntry` object. 

There's a good chance you won't get the information from the `on-click` event you need, so I'd suggest setting up a `userEntry` polymer property and passing it to `LocalUsers.createUser`. You'd also have to ensure that `userEntry` is being populated correctly, but that's a bit out of scope for `createUser` specifically.